spec/acceptance/lesson_email_spec.rb

```diff
@@ -10,13 +10,13 @@
     @teacher = FactoryGirl.create(:user)
     @attendee = FactoryGirl.create(:user)
     @venue = FactoryGirl.create(:venue)
-    @lesson = FactoryGirl.create(:next_month_lesson, start_time: Date.current + 1.month + 1.day + 19.hours, start_time: Date.current + 1.month + 1.day + 21.hours)
+    @lesson = FactoryGirl.create(:next_month_lesson, start_time: Date.current + 1.month + 1.day + 19.hours)
   end
 
   scenario "Sending notification email" do
     expect{NotificationMailer.lesson_notification(
       @lesson.id, @teacher.id, @attendee.id
-    ).deliver}.to change{ActionMailer::Base.deliveries.count}.by(1)
+    ).deliver_now}.to change{ActionMailer::Base.deliveries.count}.by(1)
```

Comment:
Expression at 19, 18 should be on its own line.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.